Sources/MapboxNavigation/NavigationMapView.swift

```diff
@@ -362,14 +367,6 @@ open class NavigationMapView: UIView {
         mapView.mapboxMap.style.removeSources(sourceIdentifiers)
     }
     
-    func removeRestrictedRouteArea() {
```

Comment:
Remove the unused code. Right now the route line update are all through the `NavigationMapView.show(_:layerPosition:legIndex:)` which includes all the previous routes removal.